ex1.html

```diff
@@ -18,53 +20,57 @@
 					<br><br>
 					<input type="password" placeholder="Password" name="pass_box" class="loginboxes">
 					<br><br><br><br><br><br><br>
-						<a href="ex2.html" class="loginbutton">Login</a>
+					<a href="ex2.html" class="loginbutton">Login</a>
 				</form> 
 			</div>
 		</div>
 	</section>
 
 	<section>
-		<div>
-			<p><img src="new-feedback.jpg" style="float:left" >
+		<div class="div1">
```

Comment:
How many `<div>`s are you expecting to have in this document? What if at one point you reach 30? 50? Maybe 100 different sections? Are you going to add a separate class for each?